src/sentry/rules/conditions/existing_high_priority_issue.py

```diff
@@ -33,19 +33,24 @@ def get_activity(
                 project=self.project,
                 datetime__gte=start,
                 datetime__lt=end,
-                type__in=[ActivityType.SET_ESCALATING.value],
+                type__in=[ActivityType.SET_PRIORITY.value],
```

Comment:
why was this escalating previously?

Reply:
oh are we making this more general? so all high priority activites will trigger this now?

Reply:
We're not generalizing the alert, but this was previous activity filter was incorrect. 

`SET_ESCALATING` is a superset of `SET_PRIORITY` since not all escalating groups have high priority. If we have an Activity entry with `SET_PRIORITY` and a priority of high though, that means the group escalated and has high priority and would trigger the alert.